Library/Location.swift

```diff
@@ -102,19 +104,19 @@ struct Location {
     }
 
     private var today: Date {
-        return Calendar.autoupdatingCurrent.startOfDay(for: Date())
+        return Date()
```

Comment:
why did this stop mattering?

Reply:
^^ still curious about this

Reply:
Nowhere is the time of the day taken into account there. The place that does need it, needs the current one, not midnight. 👍